python/llm/dev/benchmark/perplexity/run.py

```diff
@@ -44,7 +44,7 @@ def main():
     additional_model_kwargs = parse_kwargs(args.model_kwargs)
     summary = {}
     for precision in args.precisions:
-        model_kwargs = additional_model_kwargs
+        model_kwargs = additional_model_kwargs.copy()
```

Comment:
does this change impact anything?

Reply:
Without `copy()`, when the precisions include `float16` at the beginning, all following precisions will run with`float16` since this line actually modifies the original `additional_model_kwargs`

Reply:
make sense.